### PR TITLE
docs: Removed outdated Gitpod demo from README

### LINKED ISSUE #2255: Remove outdated Gitpod demo from the README
### Description

Remove the outdated Gitpod demo from the README. The demo was based on v1 of the project

Things to remove
- Heading along with the GIF

Screenshot 2022-11-21 at 4 07 17 PM

### PR DESCRIPTION
**Pull request by ShorunTech**
## Fixes Issue

fix #2255

## Changes proposed

https://github.com/EddieHubCommunity/LinkFree/issues/2255

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot (260)

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2256"></a>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,9 +17,6 @@
 
 ![Eddie Jaoude's LinkFree profile GIF](https://user-images.githubusercontent.com/106697681/176986263-b8c278f1-41a8-4b85-80e6-be46e9cc9a00.gif)
 
-## Example using Gitpod, an ephemeral dev environment in the cloud (free)
-
-![Gitpod GIF with progress bar](https://user-images.githubusercontent.com/46727048/146048451-ed4ff31a-c178-4713-a9e0-95118be742dc.gif)
```
Comment (Pradumnasaraf) [inline README.md:20]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for PR. Please remove the extra line.

**Comment by ShorunTech** [comment]
okay

File: docs/contributing/CONTRIBUTING.md [hunk 2]
```diff
@@ -1,8 +1,10 @@
+
 ## 👨‍💻 Prerequisite Skills to Contribute
 
 ### Contribute in public/Profile
 
-- [Git](https://git-scm.com/)
+
```
Comment (schmelto) [inline docs/contributing/CONTRIBUTING.md:6]:
```suggestion
```

**Comment by Pradumnasaraf** [comment]
If you need any help, do let's know @ShorunTech

**Comment by eddiejaoude** [comment]
Thank you for the contribution and collaboration. I will close this PR as the docs have moved forward and now have conflicts. 

Feel free to create a new pull request from the latest docs.

**Pull request closed without merging**